### ISSUE TITLE
[Bug Report] v-rating does not allow accessible keyboard navigation

### ISSUE THREAD

**Issue by flpehr0** [vuetifyjs/vuetify#10820]
### Environment
**Vuetify Version:** 2.2.11
**Vue Version:** 2.6.11
**Browsers:** Chrome 80.0.3987.132, Mozilla Firefox
**OS:** Windows 10

### Steps to reproduce
- Turn NVDA or JAWS on 
- Attempt to use the keyboard to navigate (tab) to the stars and select (enter) a rating. 

### Expected Behavior
- A user should be able to tab to, focus on, and select/unselect each star using only the keyboard
- The same CSS applied on :focus should also be as similar as possible to :hover
- When tabbing through each star, the screen reader should read the total number of stars, which star they are on, whether or not it is selected, and be notified when a state changes. 

### Actual Behavior
The stars are completely inaccessible to the keyboard and doesn't announce in screen readers.

### Reproduction Link
<a href="https://codepen.io/flpehr0/pen/bGdvbbX" target="_blank">https://codepen.io/flpehr0/pen/bGdvbbX</a>

### Other comments
Possible solutions include: 
- use of aria role="group" to associate them as a set
- use of aria labels on the stars to label where they are in the set
- using tabindex to allow keyboard focus
- using aria-current or aria-pressed to show selection
- expose the code for the stars to the developer so we can add custom accessibility attributes. 

Resource for aria-current and other aria tags: https://www.accessibility-developer-guide.com/examples/sensible-aria-usage/current/

**Comment by flpehr0** [comment]
Hi, any updates on this feature? Any additional information I can provide to help?

**Comment by johnleider** [comment]
No updates, but it has to move to v2.3 due to bandwidth issues.

If you have any additional questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).

**Comment by flpehr0** [comment]
Thank you!

**Comment by olegdon** [comment]
looking forward to having this functionality as soon as possible

**Comment by kyledrummond** [comment]
Agreed. This would be very helpful.
